src/amo/constants.js

```diff
@@ -138,6 +138,7 @@ export const INSTALL_SOURCE_MOST_POPULAR = 'mostpopular';
 export const INSTALL_SOURCE_SEARCH = 'search';
 export const INSTALL_SOURCE_TOP_RATED = 'rating';
 export const INSTALL_SOURCE_TRENDING = 'hotness';
+export const INSTALL_SOURCE_TAG_SHELF = 'tag-shelf-{tagName}';
```

Comment:
Because this is only used in one place, I'm not sure it's worth the complexity of having `{tagName}` in here and having to use `replace` in the component. What do you think of calling this `INSTALL_SOURCE_TAG_SHELF_PREFIX`, leaving off the `{tagName}`, and then simply appending the tag name to `INSTALL_SOURCE_TAG_SHELF_PREFIX` in the component?